core/backend/src/CloudSqlite.ts

```diff
@@ -824,23 +828,41 @@ export namespace CloudSqlite {
     }
 
     /**
-     * Initialize a cloud container to hold a Cloud SQliteDb. The container must first be created via its storage supplier api (e.g. Azure, or Google).
-     * A valid sasToken that grants write access must be supplied. This function creates and uploads an empty database into the container.
+     * Initialize a cloud container to hold VersionedSqliteDbs. The container must first be created by [[createBlobContainer]].
+     * This function creates and uploads an empty database into the container.
      * @note this deletes any existing content in the container.
      */
-    protected static async _initializeDb(args: { dbType: typeof VersionedSqliteDb, props: ContainerAccessProps, dbName: string, blockSize?: "64K" | "4M" }) {
-      const container = createCloudContainer({ ...args.props, writeable: true });
+    protected static async _initializeDb(args: { dbType: typeof VersionedSqliteDb, props: ContainerProps, dbName: string, blockSize?: "64K" | "4M" }) {
+      const container = createCloudContainer({ ...args.props, writeable: true, accessToken: args.props.accessToken ?? await CloudSqlite.requestToken(args.props) });
       container.initializeContainer({ blockSize: args.blockSize === "4M" ? 4 * 1024 * 1024 : 64 * 1024 });
       container.connect(CloudCaches.getCache({ cacheName: this._cacheName }));
       await withWriteLock({ user: "initialize", container }, async () => {
         const localFileName = join(KnownLocations.tmpdir, "blank.db");
-        args.dbType.createNewDb(localFileName);
+        args.dbType.createNewDb(localFileName, args);
         await transferDb("upload", container, { dbName: args.dbName, localFileName });
         unlinkSync(localFileName);
       });
       container.disconnect({ detach: true });
     }
 
+    /**
+     * Create a new BlobContainer from the BlobContainer service to hold one or more VersionedSqliteDbs.
+     * @returns A ContainerProps that describes the newly created container.
+     * @note the current user must have administrator rights to create containers.
+     */
+    protected static async createBlobContainer(args: Omit<BlobContainer.CreateNewContainerProps, "userToken">): Promise<CloudSqlite.ContainerProps> {
```

Comment:
There was previously never a way to create a blob container and also initialize it for a CloudSqlite use. Previously you had to create the container first, and then call "initialize" for the type of CloudSqlite client. This new capability is important for WorkspaceDbs, but i also added it for ViewStores and PropertyStores (though admittedly that has nothing to do with this PR).